### PR TITLE
fix(DB/Quest):Corruption of Earth and Seed repeatable after faction c…

### LINKED ISSUE #7273: Faction change and reward: Corruption of Earth and Seed
### Current Behaviour

Can be completed twice after faction change.

Rewards 7 reputation for Cenerion Circle.

### Expected Blizzlike Behaviour

Rewards 500 reputation.

Should be exclusive to the other quest.

Related: https://github.com/azerothcore/azerothcore-wotlk/issues/5592

### Source

https://classic.wowhead.com/quest=7065/corruption-of-earth-and-seed

classic says 150 rep

https://wowpedia.fandom.com/wiki/Corruption_of_Earth_and_Seed_(quest)

tbc db/pedia from pre-cata says 1000 rep

https://wotlk.evowow.com/?quest=7065

other 3.3.5 databases have it as 500 rep

### Steps to reproduce the problem

1. start quest 7064
2. complete quest (observe rep reward)
3. faction change
4. start quest 7065
5. complete quest 

### Extra Notes

fix included

fix may not be valid/correct as not 100% sure on exlusivegroup SQL functionality + first fix by reporter in keira3

```
faction change:
UPDATE quest_template_addon SET ExclusiveGroup = 7065 WHERE (ID = 7064);
UPDATE quest_template_addon SET ExclusiveGroup = 7064 WHERE (ID = 7065);
rewards:
UPDATE quest_template SET RewardFactionValue1 = 500 WHERE (ID = 7065);
UPDATE quest_template SET RewardFactionValue1 = 500 WHERE (ID = 7064);
```

### AC rev. hash/commit

chromiecraft/azerothcore-wotlk@36a1777

### Operating system

win10

### Custom changes or Modules

    mod-ah-bot
    mod-cfbg
    mod-chromie-xp
    mod-desertion-warnings
    mod-duel-reset
    mod-eluna-lua-engine
    mod-ip-tracker
    mod-low-level-arena
    mod-multi-client-check
    mod-pvp-titles
    mod-pvpstats-announcer
    mod-queue-list-cache
    mod-server-auto-shutdown
    lua-CarbonCopy
    lua-CustomWorldboss
    lua-LevelUpReward
    lua-send-and-bind
    lua-Zonecheck
    lua-TempAnnouncements

### PR DESCRIPTION
**Pull request by baranzadeoglu**
…change

Puts quest ids 7064 and 7065 into same exclusive group as suggested by the relevant documentation.
Fixes #7273
Note that the reputation issue in #7273 is a replication of a visual bug on Keira3 and should be ignored.

## Changes Proposed:
-  Put quest ids 7064 and 7065 into the same exclusive group 7064.
-  

## Issues Addressed:

- Closes  #7273  https://github.com/chromiecraft/chromiecraft/issues/1372

## Tests Performed:

- 
- 

## How to Test the Changes:

1. complete either quest
2.change faction
3.try to complete the other

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1631992050760579041.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1631992050760579041');
+
+UPDATE quest_template_addon SET ExclusiveGroup = 7064 WHERE (ID = 7064);
+UPDATE quest_template_addon SET ExclusiveGroup = 7064 WHERE (ID = 7065);
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1631992050760579041.sql:4]:
```suggestion
UPDATE quest_template_addon SET ExclusiveGroup = 7064 WHERE (ID IN (7064,7065));
```

File: data/sql/updates/pending_db_world/rev_1631992050760579041.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1631992050760579041');
+
+UPDATE quest_template_addon SET ExclusiveGroup = 7064 WHERE (ID IN (7064,7065));
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1631992050760579041.sql:3]: backticks

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Review by Helias — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1631992050760579041.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1631992050760579041');
+
+UPDATE 'quest_template_addon' SET 'ExclusiveGroup' = '7064' WHERE ('ID' IN (7064,7065));
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1631992050760579041.sql:3]:
```suggestion
UPDATE `quest_template_addon` SET `ExclusiveGroup` = 7064 WHERE (`ID` IN (7064,7065));
```
Use backticks \` \` and dont use single quotes around non string values

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Comment by FrancescoBorzi** [comment]
image

@baranzadeoglu can you please specify what tests have you made with your code

**Comment by baranzadeoglu** [comment]
> @baranzadeoglu can you please specify what tests have you made with your code

none.

**Comment by stoudtlr** [comment]
Test results:
1. Completed quest id 7065 as alliance and received 500 rep
2. Faction change and relog
3. completed quest id 7064 as horde and received another 500 rep

image

**Comment by baranzadeoglu** [comment]
> Test results:
> 
>     1. Completed quest id 7065 as alliance and received 500 rep
> 
>     2. Faction change and relog
> 
>     3. completed quest id 7064 as horde and received another 500 rep
This is strange. I am not an expert or anything with exclusive groups but from the Wiki,

>    if ExclusiveGroup > 0
>
>Allows to define a group of quests of which only one may be chosen and completed. E.g. if from quests 1200, 1201 and 1202 >only one should be allowed to be chosen, insert 1200 into ExclusiveGroup of all 3 quests.

**Comment by FrancescoBorzi** [comment]
what's the status of this?

**Comment by baranzadeoglu** [comment]
> what's the status of this?

this fix does not seem to work although it is in line with the wiki. To best of my understanding there is something above me going on.

**Comment by acidmanifesto** [comment]
What is the status of this PR? is it abandoned now?

**Comment by baranzadeoglu** [comment]
> What is the status of this PR? is it abandoned now?

well, the documentation for the command is incorrect. I am waiting for someone to figure out what exactly is wrong with the documentation.

**Comment by Kitzunu** [comment]
This is wrong anyways. https://github.com/azerothcore/azerothcore-wotlk/pull/11403 is correct fix

**Pull request closed without merging**